modules/serverless-logic-upgrade-1.36-scaling-up-workflows-with-gitops-profile.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1.36-scaling-up-workflows-with-gitops-profile_{context}"]
+= Scaling up workflows with the gitops profile
+
+Workflows with the `gitops` profile that were previously scaled down must be scaled back up to continue operation.
+
+.Procedure
+
+. Modify the `my-workflow.yaml` CRD and scale up each workflow to `one` before upgrading as shown in the following example:
++
+[source,yaml]
+----
+spec:
+  podTemplate:
+    replicas: 1
+----
+
+. Apply the updated CRD by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <my-workflow.yaml> -n <target_namespace>
+----
+
+. (Optional) Scale the workflow back to `1` by running the following command:
```

Comment:
```suggestion
. (Alternative) Scale the workflow back to `1` by running the following command:
```

Reply:
@wmedvede 
According to the IBM & OpenShift standard, we have to use (Optional) here.